tests/conftest.py

```diff
@@ -1,11 +1,50 @@
-import aiohttp
+# Licensed to CRATE Technology GmbH ("Crate") under one or more contributor
+# license agreements.  See the NOTICE file distributed with this work for
+# additional information regarding copyright ownership.  Crate licenses
+# this file to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# However, if you have executed another commercial license agreement
+# with Crate these terms will supersede the license and you may use the
+# software solely pursuant to the terms of the relevant commercial agreement.
+
+from unittest import mock
+
 import pytest
+import urllib3
+
+from croud.api import Client
 
-from tests.util.fake_server import FakeCrateDBCloud, FakeResolver
+from .util.fake_cloud import FakeCrateDBCloud
+
+
+@pytest.fixture(scope="session")
+def fake_cratedb_cloud():
+    cloud = FakeCrateDBCloud()
+    cloud.start()
+    yield cloud
+    cloud._server.shutdown()
 
 
 @pytest.fixture
-async def fake_cloud_connector(event_loop):
-    async with FakeCrateDBCloud(loop=event_loop) as dns_info:
-        resolver = FakeResolver(dns_info, loop=event_loop)
-        yield aiohttp.TCPConnector(loop=event_loop, resolver=resolver, ssl=True)
+def client(fake_cratedb_cloud):
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+    with mock.patch(
+        "croud.api.cloud_url",
+        return_value=f"https://127.0.0.1:{fake_cratedb_cloud.port}",
+    ):
+        with mock.patch("croud.api.get_verify_ssl", return_value=False):
+            with mock.patch(
+                "croud.api.Configuration.get_token", return_value="some-token"
+            ):
+                yield Client(env="local", region="bregenz.a1")
```

Comment:
As described above, I would use a dedicated `verify_ssl` keyword instead:

```suggestion
                yield Client(env="local", region="bregenz.a1", verify_ssl=False)
```